### ISSUE TITLE
Wrong resources in output

### ISSUE THREAD

**Issue by hsychla** [jdockerty/kubectl-oomd#20]
I was checking some OOM'd pods this morning and at least one of them is showing wrong values in the REQUEST and LIMIT columns.
While the restarted container is configured with `memory:  6Gi` for both request and limit, the output shows `64Mi / 128Mi`.

I _assume_ that you are fetching the values from another container in the pod since there are some with those values (the pod has 8 containers).

**Comment by jdockerty** [comment]
That's quite interesting.

I wonder whether it has to do with the ordering of key-values when there are a load of pods. I suspect that is the case, as I've seen it work as expected for pods which contain at-most 3 containers.

I'll try replicate this with creating a local pod with a bunch of containers and see whether I get some mismatches on the `REQUEST` and `LIMIT` columns.

Thanks for pointing this out!

**Comment by jdockerty** [comment]
@hsychla As another quick check, similar to before :smile: 

Could you run `make bin` and `bin/oomd` from the `fix/request-limits-with-lots-of-containers` branch of this repository please :blush: 

I was able to recreate the bug, I assume your output would look similar to this with the mismatched requests?

```
POD                        CONTAINER         REQUEST     LIMIT     TERMINATION TIME
oomer-57874f78f8-zn4wn     oomer-100-100     128Mi       128Mi     2022-12-08 17:36:47 +0000 GMT
oomer-57874f78f8-zn4wn     oomer-100-200     50Mi        50Mi      2022-12-08 17:37:12 +0000 GMT
oomer-57874f78f8-zn4wn     oomer-128-128     100Mi       100Mi     2022-12-08 17:36:27 +0000 GMT
```

In this example, I added the limit/request to the container name to demonstrate the mismatch, e.g. `oomer-100-100` should be the `100Mi` columns.

**Comment by hsychla** [comment]
Hi,

ran a quick test and it looks like it fixes the issue.
Thanks for the quick turnaround!

**Comment by jdockerty** [comment]
Brilliant, much appreciated for testing that for me 😊 

I'm going to add an extra test to make sure the limit/request actually matches what is defined in the manifest. Once that's done and I've merged the PR I'll likely do a release for `0.0.7` and then this will be fixed on the `krew` version too.

**Comment by hsychla** [comment]
Cool!
I also saw that you added sort by date. Very happy about that as well!
